ci-operator/config/openshift/cluster-api/openshift-cluster-api-openshift-master.yaml

```diff
@@ -0,0 +1,54 @@
+base_images:
+  base:
+    name: "4.10"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.16-openshift-4.10:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.16-openshift-4.10
+binary_build_commands: make managers
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+canonical_go_repository: sigs.k8s.io/cluster-api
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: cluster-api
+promotion:
```

Comment:
In hypershift we want this to promote to 
```
promotion:
  namespace: hypershift
  tag: latest
```

Reply:
Yes, we will need different files